extensions/ai-translator/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "ai-translator",
+  "title": "AI Translator",
```

Comment:
The name doesn't quite describe what it is. Could we consider using a different name?